CHANGES.md

```diff
@@ -503,6 +503,8 @@
 
     - Added `logTitles` to log all window titles (focused and unfocused ones).
 
+    - Added `logTitlesOnScreen` as a screen-specific variant of `logTitles`.
```

Comment:
Should we merge these two entries together?

Reply:
Oh yeah we definitely should, thanks